airflow/models/taskinstance.py

```diff
@@ -94,7 +94,6 @@
 from airflow.models.dagbag import DagBag
 from airflow.models.dataset import DatasetAliasModel, DatasetModel
 from airflow.models.log import Log
-from airflow.models.mappedoperator import MappedOperator
```

Comment:
Why do we need to make this change?

Reply:
It is throwing partial import error.

Reply:
Do you know what caused the partial import error? Would it be possible for us to extract that as a separate module? if not, I'm also good with the current implementation

Reply:
This was not my change. This was change from previous PRs from main.